smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -116,5 +116,9 @@ void shouldSendTelemetry() throws InterruptedException {
         .as("Should instrument logs")
         .startsWith("Starting ")
         .contains(this.getClass().getSimpleName());
+    String codeNamespace = firstLog.getAttributes().get(SemanticAttributes.CODE_NAMESPACE);
```

Comment:
IIUC, this test makes sure the "reinit" logic works - do we have a test that also tests that an appender gets added?